examples/protocol_specification_ex/output/two_party_negotiation/message.py

```diff
@@ -163,35 +124,46 @@ def query(self) -> DataModel:
         return cast(DataModel, self.get("query"))
 
     @property
-    def price(
-        self,
-    ) -> Optional[
-        Union[
-            Dict[int, Unit],
-            FrozenSet[int],
-            Tuple[Ali],
-            bool,
-            str,
-            Dict[_IOTApp, bytes],
-            Dict[DataModel, bool],
-        ]
-    ]:
+    def number(self) -> int:
```

Comment:
Can we sort the properties by alphabet? I.e. first the block of meta-data ones by alphabet, then the ones specific to the protocol by alphabet.

Reply:
Done